apps/explorer/src/pages/transaction-result/TransactionView.tsx

```diff
@@ -417,10 +419,23 @@ function TransactionView({
     const txError = getExecutionStatusError(transaction);
 
     const gasData = getGasData(transaction);
-
     const gasPrice = gasData.price || 1;
     const gasPayment = gasData.payment;
     const gasBudget = gasData.budget;
+    const gasOwner = gasData.owner;
+    const isSponsoredTransaction = gasOwner !== sender;
+
+    const transactionSignatures = getTransactionSignature(transaction);
+    const deserializedTransactionSignatures = transactionSignatures.map(
+        (signature) => fromSerializedSignature(signature)
+    );
+    const accountSignature = getSignatureFromAddress(
+        deserializedTransactionSignatures,
+        sender
+    );
+    const sponsorSignature = isSponsoredTransaction
+        ? getSignatureFromAddress(deserializedTransactionSignatures, gasOwner)
+        : undefined;
```

Comment:
why not null

Reply:
because I'm bad at differentiating sometimes 😆, will fix this!